app/org/sagebionetworks/bridge/models/ResourceList.java

```diff
@@ -0,0 +1,18 @@
+package org.sagebionetworks.bridge.models;
+
+import java.util.List;
+
+public final class ResourceList<T> {
```

Comment:
Add documentation that this is creating a JSON payload wrapper around a list, and keeps the door open to adding parameters for searching, paging, filtering.